docs/source/science_examples.rst

```diff
@@ -6,12 +6,12 @@ Science Examples
   :caption: Science Examples:
 
 
-  science/HLS/HLS_search.ipynb
-  science/GEDI/GEDI02_A_tutorial.ipynb
-  science/GEDI/SearchGEDI.ipynb
+  science/HLS/HLS30.ipynb
+  science/GEDI/GEDI_L2A.ipynb
+  science/GEDI/GEDI_L2B.ipynb
   science/GEDI/GEDI_L3_tutorial.ipynb
   science/GEDI/GEDI_L4A_Example.ipynb
   science/GEDI/GEDI_L4B.ipynb 
-  science/ATL03/IceSat_ATL03_Tutorials.ipynb
-  science/NISAR/Nisar_access_explore.ipynb
+  science/ATL03/ATL03.ipynb
+  science/NISAR/Nisar_access.ipynb
   science/AfriSAR_AGB_1681/AfriSAR_AGB_Maps_1681.ipynb
```

Comment:
Is this one that's going to be added to the website? I noticed it on the ADE but haven't made any edits yet. Looks like it'll need some extra work, especially when it comes to VEDA guidelines, and the collection may need to be updated since it looks like the one in the tutorial doesn't exist in STAC.

Reply:
STAC is ongoing work, the current uses a direct URL. Yes it should be on the site even though it's early stages. I'm clarifying it's nisar_simulated